### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by Zach-Johnson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## 2.3.0 (Unreleased)
 
+## 2.2.1 (Febuary 14, 2020)
+BUG FIXES
+- Resolves a panic and several type issues with the ASN field.
```
Comment (mburtless) [inline CHANGELOG.md:5]: Don't some IP whitelist fixes also land in this version?

**Review by mburtless — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Zach-Johnson) [reply CHANGELOG.md:5]: There were some ip whitelist fixes included in 2.2.0, not sure if that's what you are referring to?

**Comment by mburtless** [comment]
Whoops, sorry didn't realize those hadc already landed.

File: CHANGELOG.md (hunk 1 above)
Comment (mburtless) [reply CHANGELOG.md:5]: D'oh that was it. Forgot that made it into 2.2.0

**Review by mburtless — APPROVED** [review]

**Pull request merged**